var/spack/repos/builtin/packages/survey/package.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Survey(CMakePackage):
+    """Survey is a high-level performance tool product from Trenza, Inc.
+       The survey collector/analytics framework is a new generation,
+       high-level, light-weight multi-platform Linux tool set that
+       targets metric collection for high-level performance analysis
+       of applications running on both single node and on large-scale
+       platforms, including the Cray platforms.
+
+       The collector is designed to work on sequential, MPI, OpenMP,
+       and hybrid codes and directly leverages several interfaces
+       available for tools inside current MPI implementations including:
+       MPICH, MVAPICH, MPT, and OpenMPI. It also supports multiple
+       architectures and has been tested on machines based on Intel,
+       AMD, ARM, and IBM P8/9 processors and integrated GPUs.
+    """
+
+    # To access the survey source and build with
+    # spack please contact: Trenza Inc. via:
+    # dmont@trenzasynergy.com
+    homepage = "http://www.trenzasynergy.com"
+    git      = "git@gitlab.com:trenza/survey.git"
+
+    maintainers = ['jgalarowicz']
+
+    version('develop', branch='master')
+    version('1.0.0', branch='1.0.0')
+
+    variant('mpi', default=False,
+            description="Enable mpi, build MPI data collector")
+
+    variant('build_type', default='RelWithDebInfo',
+            description='The build type to build',
+            values=('Debug', 'Release', 'RelWithDebInfo'))
```

Comment:
Isn't this a duplicate of what's already inherited from `CMakePackage`?

Reply:
I wasn't aware of that.  I will remove and check in soon.